bin/phoenix_utils.py

```diff
@@ -77,10 +77,13 @@ def findClasspath(command_name):
     return tryDecode(subprocess.Popen(command, shell=True, stdout=subprocess.PIPE).stdout.read())
 
 def setPath():
-    PHOENIX_CLIENT_JAR_PATTERN = "phoenix-client-hbase-*[!s].jar"
+    PHOENIX_CLIENT_EMBEDDED_JAR_PATTERN = "phoenix-client-embedded-hbase-*[!s].jar"
```

Comment:
phoenix-client-embedded is missing the SLF4J backend as well.
It should handled similarly to the sqlline JAR, with loading it from /lib by default, but having an option to override (and use a different backed JAR)